runtime/doc/eval.txt

```diff
@@ -2477,6 +2478,28 @@ call({func}, {arglist} [, {dict}])			*call()* *E699*
 		{dict} is for functions with the "dict" attribute.  It will be
 		used to set the local variable "self". |Dictionary-function|
 
+capture({command})					*capture()*
+		Capture output of {command}.
+		If {command} is a |String|, it returns {command} output.
+		If {command} is a |List|, it returns concatenated outputs of
+		each item.
+		Examples: >
+			let a = capture('echon "foo"')
+			echo a
+<			foo >
+			let a = capture(['echon "foo"', 'echon "bar"'])
+			echo a
+<			foobar
+		This function is not available in the |sandbox|.
+		Note: The commands are run as if they were prepended with
+		|:silent| modifier.  |:redir| and |capture()| work in parallel:
```

Comment:
"work in parallel" => "work together"

mention of "parallel" could be confusing because evokes the idea of parallelism.